### PR TITLE
Update amldecoder.cpp

### PR DESCRIPTION
**Pull request by persianpros**

### CODE REVIEW

File: lib/dvb/amldecoder.cpp [hunk 1]
```diff
@@ -249,6 +249,10 @@ RESULT eAMLTSMPEGDecoder::setAudioPID(int apid, int type)
 				m_codec.audio_type = AFORMAT_AC3;
 				t = "AC3";
 				break;
+			case aDDP:
+				m_codec.audio_type = AFORMAT_EAC3;
```
Comment (WanWizard) [inline lib/dvb/amldecoder.cpp]: where is AFORMAT_EAC3 defined?

Comment (WanWizard) [inline lib/dvb/amldecoder.cpp:253]: where is AFORMAT_EAC3 defined?

**Comment by pieterg** [comment]
Can't we just remove all this terrible #if AML stuff which adds non-linuxdvb api workarounds?
It should never have been accepted into e2 in the first place if you ask me...

Is anyone still using it?

**Comment by mirakels** [comment]
This was needed to support Wetek boxes. 
As Wetek apparently is not supported anymore this code could be removed.

**Comment by littlesat** [comment]
+1

**Pull request closed without merging**